src/electron/get_electron_build_parameters.mjs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {getBuildParameters} from '../build/get_build_parameters.mjs';
+
+const ELECTRON_PLATFORMS = ['linux', 'windows'];
+
+/*
+  Inputs:
+  => platform: the list of action arguments passed in
+
+  Outputs:
+  => an object containing the required electron parameters.
+*/
+export function getElectronBuildParameters(parameters) {
+  const {platform, buildMode, stagingPercentage, networkStack, sentryDsn} = getBuildParameters(parameters);
+
+  if (!ELECTRON_PLATFORMS.includes(platform)) {
```

Comment:
This is a bit of an [anemic model](https://en.wikipedia.org/wiki/Anemic_domain_model). Not needed, but you may consider using OO design instead, with a Platform type that has a isElectron() method.

Reply:
Yeah, to be honest, I've been approaching these scripts thinking they're all going to be thrown out anyway. This is a bigger refactor we can do if we decide we want to keep them, but I'm hoping they become unnecessary once we move to a monorepo.